### PR TITLE
feat(common_awsim_labs_sensor_launch): filter load from parameter file

### PR DESCRIPTION
**Pull request by vividf**
## Description

- https://github.com/autowarefoundation/autoware.universe/pull/8469 removes the default parameters for filter.cpp.
Thus, the launcher needs to load the filter parameter from the parameter file

Related:
- https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/104

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by vividf** [comment]
@xmfcx could you reveiw this, thanks!

File: common_awsim_labs_sensor_launch/config/filter.param.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**:
+  ros__parameters:
+    input_frame: ""
+    output_frame: ""
+    max_queue_size: 5
+    use_indices: false
+    latched_indices: false
+    approximate_sync: false
```
Comment (xmfcx) [inline common_awsim_labs_sensor_launch/config/filter.param.yaml:8]: @vividf this has a weird line ending, could you check it?

It seems fine in:
- https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/104

**Review by xmfcx — CHANGES_REQUESTED** [review]

File: common_awsim_labs_sensor_launch/config/filter.param.yaml (hunk 1 above)
Comment (xmfcx) [reply common_awsim_labs_sensor_launch/config/filter.param.yaml:8]: image

Comment (vividf) [reply common_awsim_labs_sensor_launch/config/filter.param.yaml]: Thanks, fixed it!

**Comment by xmfcx** [comment]
I will look into the precommit problem later on.

Is this ready to be merged now?

**Which PR from the universe is this connected to?**

**Review by xmfcx — APPROVED** [review]

**Comment by xmfcx** [comment]
Sorry it's this one:
- https://github.com/autowarefoundation/autoware.universe/pull/8380

I've approved this.

First universe PR should be merged.
Then we can merge these.

**Comment by vividf** [comment]
I'll close this one for now since it makes the nodes harder to maintain.

**Pull request closed without merging**